dept44-api-models/src/main/java/se/sundsvall/dept44/api/models/paging/validation/impl/ValidSortByPropertyConstraintValidator.java

```diff
@@ -0,0 +1,47 @@
+package se.sundsvall.dept44.api.models.paging.validation.impl;
+
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+import jakarta.persistence.Column;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.hibernate.validator.internal.engine.messageinterpolation.util.InterpolationHelper;
+import se.sundsvall.dept44.api.models.paging.AbstractParameterPagingBase;
+import se.sundsvall.dept44.api.models.paging.validation.ValidSortByProperty;
+
+import java.lang.reflect.Field;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+
+public class ValidSortByPropertyConstraintValidator implements ConstraintValidator<ValidSortByProperty, AbstractParameterPagingBase> {
+
+	private static final String CUSTOM_ERROR_MESSAGE_TEMPLATE = "One or more of the sortBy members %s are not valid. Valid properties to sort by are %s.";
```

Comment:
"One or more of the sortBy members %s are not valid...."
->
"One or more of the sortBy properties %s are not valid...."